pkg/workloads/cortex/lib/api_utils.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2019 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from cortex.lib.exceptions import UserException, CortexException
+from cortex.lib.log import get_logger
+
+logger = get_logger()
+
+
+def api_metric_dimensions(ctx, api_name):
+    api = ctx.apis[api_name]
+    return [
+        {"Name": "AppName", "Value": ctx.app["name"]},
+        {"Name": "APIName", "Value": api["name"]},
+        {"Name": "APIID", "Value": api["id"]},
+    ]
+
+
+def status_code_metric(dimensions, status_code):
+    status_code_series = int(status_code / 100)
+    dimensions.append({"Name": "Code", "Value": "{}XX".format(status_code_series)})
```

Comment:
If instead of appending we made a copy here, could we remove the `copy()`s below?